lib/mayaUsdAPI/python/wrapProxyShapeNotice.cpp

```diff
@@ -21,102 +21,106 @@
 
 #include <boost/python/def.hpp>
 
-PXR_NAMESPACE_OPEN_SCOPE
+#include <assert.h>
+
+namespace {
 
 // Listen to mayaUsdAPI stage notices
-class StageNoticesListener : public TfWeakBase
+class StageNoticesListener : public PXR_NS::TfWeakBase
 {
 public:
     StageNoticesListener()
     {
-        TfSingleton<StageNoticesListener>::SetInstanceConstructed(*this);
-        TfWeakPtr<StageNoticesListener> ptr(this);
-        _stageSetKey = TfNotice::Register(ptr, &StageNoticesListener::_StageSet);
-        _stageInvalidateKey = TfNotice::Register(ptr, &StageNoticesListener::_StageInvalidate);
+        PXR_NS::TfSingleton<StageNoticesListener>::SetInstanceConstructed(*this);
+        PXR_NS::TfWeakPtr<StageNoticesListener> ptr(this);
+        _stageSetKey = PXR_NS::TfNotice::Register(ptr, &StageNoticesListener::_StageSet);
+        _stageInvalidateKey
+            = PXR_NS::TfNotice::Register(ptr, &StageNoticesListener::_StageInvalidate);
         _stageObjectsChangedKey
-            = TfNotice::Register(ptr, &StageNoticesListener::_StageObjectsChanged);
-        TF_VERIFY(
+            = PXR_NS::TfNotice::Register(ptr, &StageNoticesListener::_StageObjectsChanged);
+        assert(
             _stageSetKey.IsValid() && _stageInvalidateKey.IsValid()
             && _stageObjectsChangedKey.IsValid());
     }
 
     ~StageNoticesListener() { _Revoke(); }
 
     // StageSet notice
-    bool StageSetHasBeenCalled() const { return _stageSetCalled; }
-    void ResetStageSet() { _stageSetCalled = false; }
+    bool StageSet() const { return _bStageSet; }
+    void ResetStageSet() { _bStageSet = false; }
```

Comment:
According to our coding standard:
- Function/Method Names All functions should be lowerCamelCase.
- Variable Names Variables names, including function parameters and data members should use lowerCamelCase

In Ufe we used a trailing _ in the method name, so maybe you could have:

```
    bool stageSet const { return _stageSet; }
private:
    bool _stageSet;
    bool stageSet_(const MayaUsdAPI::ProxyStageSetNotice& notice) { _stageSet = true; }    
```